src/orga/governance.rst

```diff
@@ -120,21 +121,27 @@ To call for a standard vote, here is a template PR comment:
          * vote description
          * vote policy
          * vote totals
-         * Poll open and close dates
-         
-         If a vote has "timed out," as described in "Quorum," the documentation regarding
-         notification records should also be in this file.
-         
+         * poll open and close dates
+         * notifications given to the core group         
 
 ----
-:Quorum: All percentages below express *both* required participation, as a
+:Quorum: Quorum for votes can be met in one of three ways depending on the vote: the standard quorum rules, 
+         the accelerated quorum rules, and the "time out" quorum rules. The specific quorum rules applicable 
+         for each vote are listed below.
+
+         **Standard Quorum Rules**: All percentages below express *both* required participation, as a
          fraction of the active core team, as well as the fraction of that
          fraction who vote affirmatively on the issue. For example, in a vote
          requiring 50%, with 18 active core members, at least 9 must vote;
          if 9 vote, there must 5 affirmative votes. If 13 members vote, 7
          must be affirmative.
+         
+         **Accelerated Quorum Rules**: For certain votes, we allow a lower quorum level. For these votes,
+         if the vote ends after at least a week and there are no "no" votes, a quorum of half of the size 
+         needed for a **standard quorum** above is accepatble. For example, for a vote requiring 50% with 18
+         active core members, at least 5 people must vote "yes" and exactly 0 people must vote "no".
```

Comment:
Apache is just 3 votes. Do we feel that 3 votes is too small a group?

Reply:
I implemented your comment here exactly:

> There was a discussion in the conda-tools slack about making voting simpler by adopting an apache-esque voting system where you only need half of the original vote requirements as long as no one votes "no". I'll submit a PR to our governance docs with those changes (at some point)

I think we could go all the way that way, but it might be nice to relax just a little at first to see how it goes?